base/array.jl

```diff
@@ -833,7 +833,7 @@ end
 function indcopy(sz::Dims, I::Vector)
     n = length(I)
     s = sz[n]
-    for i = n+1:length(sz)
+    for i = n+1:length(sz) #Fixme iter, might be unnecessary, because it is not exported
```

Comment:
This is OK here since `I` is restricted to `Vector` (as opposed to `AbstractVector`).